dev/config.yaml

```diff
@@ -22,7 +25,8 @@ global:
   cloudProvider: ""  # For local development, options ("", "aws", "gcp", "azure")
   storageClass: "local-storage"  # For local development, options ("local-storage", "")
   authMethod: "password"
-  datafoldRepository: "us-docker.pkg.dev/datadiff-mm/datafold/datafold-server-dev"
+  datafoldRepository: "us-docker.pkg.dev/datadiff-mm/datafold/server"
+  shellRepository: "us-docker.pkg.dev/datadiff-mm/datafold/server"
```

Comment:
What is the point of this change?

Reply:
@cfernhout 
So I didn't want to expose our DEV image that we have. Locally in my overrides, I am overriding the shellRepository with datafold-server-dev. That is the ONLY container that should run the DEV code in dev at least. That is because that container is the one that contains the rust rebuild part that we probably need in CI, but only starting the dev stack.

Reply:
If you are deploying the prod image, why would you need the dev image dfshell? You don't have it on SaaS or any Single Tenant deployment. Why would you need it here?

Reply:
Because Rust is rebuilt in dfshell. It doesnt get rebuilt in prod. We mount everything in the dev stack, so that when rust code is touched, it gets rebuilt, deployed and then restarting the sever. Without that dev image, dfshell does nothing and server hangs.